rmc-docs/build-docs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd $SCRIPT_DIR
+
+# Download mdbook release (vs spending time building it via cargo install)
+FILE="mdbook-v0.4.12-x86_64-unknown-linux-gnu.tar.gz"
+URL="https://github.com/rust-lang/mdBook/releases/download/v0.4.12/$FILE"
+if [ ! -x mdbook ]; then
+    wget -O "$FILE" "$URL"
```

Comment:
Can we download this in a folder outside of the script directory?

Reply:
There isn't really anyplace better to put it that I can think of.

Reply:
You could put it in the current directory, temp or take the directory as a parameter. It's not a blocker but a nice to have. It keeps the repository clean.